src/Controller/LocalGovMenuLinkGroupListBuilder.php

```diff
@@ -44,7 +44,7 @@ public function buildHeader() {
    * {@inheritdoc}
    */
   public function buildRow(EntityInterface $group) {
-
+    /** @var \Drupal\localgov_menu_link_group\Entity\LocalGovMenuLinkGroup $group */
```

Comment:
add this for phpstan to understand the type return buildRow requires EntityInterface but inspecting the types for `$group` LocalGovMenuLinkGroup is returned. This comment helps phpstan and IDEs understand what is then passed further down

Reply:
assert() works the same and is nicer though ;-)

Reply:
Just discussing in Merge Tuesday, and we acknowledge that Drupal Core uses these sort of comments, but we think even core is moving towards using PHP assert(). 

Could we change this please @millnut ?

Reply:
👍 this has now been updated